### DISCUSSION TITLE
docs/guide/composition/instance/MCDReforged/

### DISCUSSION THREAD

**Discussion by giscus** [iPanelDev/docs#43] [Giscus]
# docs/guide/composition/instance/MCDReforged/

要求

https://ipaneldev.github.io/docs/guide/composition/instance/MCDReforged/

**Comment by aurumple** [comment]
配置文件要怎么改？页面不见了

**Reply by aurumple** [reply]
找到了，竟然在plugins文件夹里，和其他插件都不一样：（
